# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#Define the result
sensor_msgs/Image goal_image
float64 decision_score
bool status
